invenio_app_ils/config.py

```diff
@@ -429,6 +436,34 @@ def _(x):
         update_permission_factory_imp=deny_all,
         delete_permission_factory_imp=deny_all,
     ),
+    keyid=dict(
+        pid_type=KEYWORD_PID_TYPE,
+        pid_minter=KEYWORD_PID_MINTER,
+        pid_fetcher=KEYWORD_PID_FETCHER,
+        search_class=KeywordSearch,
+        record_class=Keyword,
+        record_serializers={
+            'application/json': ('invenio_records_rest.serializers'
+                                 ':json_v1_response'),
+        },
+        search_serializers={
+            "application/json": (
+                "invenio_records_rest.serializers:json_v1_search"
+            )
+        },
+        item_route="/keywords/<{0}:pid_value>".format(
+            _KEYID_CONVERTER
+        ),
+        list_route="/keywords/",
+        default_media_type="application/json",
+        max_result_window=_RECORDS_REST_MAX_RESULT_WINDOW,
+        error_handlers=dict(),
+        list_permission_factory_imp=backoffice_permission,
+        read_permission_factory_imp=record_read_permission_factory,
+        create_permission_factory_imp=record_create_permission_factory,
+        update_permission_factory_imp=record_update_permission_factory,
```

Comment:
If you allow editing, you have to ensure that you re-index corresponding documents that use that keyword. `deny_all` here too?
Maybe adding a comment on both explaining why it is not allowed...